pkg/agent/datamodel/types_test.go

```diff
@@ -53,6 +53,7 @@ func TestGetCustomEnvironmentJSON(t *testing.T) {
 	}
 }
 
+//nolint:funlen,nolintlint
```

Comment:
maaaaybe also worth to have funlen ignore *_test.go files?

curious what others think

Reply:
I agree, and we actually already had it in our exclude-rules - 
```
- path: "_test\\.go"
      linters:
        - bodyclose
        - dupl
        - funlen
        - goconst
        - gosec
        - noctx
        - wrapcheck
```
I had just removed it during CLI testing! I removed the //nolints :)